post/views.py

```
@@ -0,0 +1,260 @@
+    @swagger_auto_schema(
+        operation_id="좋아요 토글",
+        operation_description="좋아요를 토글합니다. 이미 좋아요가 눌려있으면 취소합니다.",
+        request_body=SignInRequestSerializer,
+        responses={200: PostSerializer, 404: "Not Found", 400: "Bad Request"},
+    )
+    def post(self, request, post_id):
+
+        ### 1 ###
+        try:
+            post = Post.objects.get(id=post_id)
+        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)